cmssw-tool-conf.spec

```diff
@@ -1,4 +1,4 @@
-### RPM cms cmssw-tool-conf 53.0
+### RPM cms cmssw-tool-conf 53.1
```

Comment:
@smuzaffar is this the right way to have different `cmssw-tool-conf` files for CMSSW 13.0.x vs 13.1.x ?

Reply:
Normally I just increment the major version when a new tool is added

Reply:
Yes... but `54.0` is already used in 13.1.x, which adds tools I think we do not want to backport to 13.0.x.

So I've used `55.0` for the 13.1.x branch, and `53.1` for the 13.0.x branch.

Reply:
ah ok,  sorry I did not realize that it is 13.0.x PR. yes `53.1` seems right